core/src/main/scala/kafka/log/LogCleaner.scala

```diff
@@ -632,6 +632,7 @@ private[log] class Cleaner(val id: Int,
     var position = 0
     var offset = segment.baseOffset
     val maxDesiredMapSize = (map.slots * this.dupBufferLoadFactor).toInt
+    info("Updating offset map for segment %s of partition %s...".format(segment.baseOffset, topicAndPartition))
```

Comment:
The other logging lines use log.name or segment.log.file.getName instead of topicAndPartition. It might be more accurate to say something like:

```scala
info(s"Updating offset map for segment ${segment.log.file.getName}")
```

I am not positive which would be more useful in debugging. But thats another option.